deploy/operator/README.md

```diff
@@ -105,7 +105,17 @@ export ASSISTED_AGENT_CLUSTER_INSTALL_NAME=assisted-agent-cluster-install
 export ASSISTED_INFRAENV_NAME=assisted-infra-env
 export ASSISTED_PULLSECRET_NAME=assisted-pull-secret
 export ASSISTED_PRIVATEKEY_NAME=assisted-ssh-private-key
-export SPOKE_CONTROLPLANE_AGENTS=1  # currently only single-node is supported
+export SPOKE_CONTROLPLANE_AGENTS=1  # either 1 (single-node) or 3 is supported
+export SPOKE_WORKER_AGENTS=0 # Number of worker nodes
+
+# JSON encoded dicionary of machine config pool per host.  Example: '{"ostest-extraworker-5":"infra","ostest-extraworker-3":"storage"}'
+export MACHINE_CONFIG_POOLS=machine-config-pools
```

Comment:
```suggestion
export MACHINE_CONFIG_POOLS=<machine-config-pools>
```

otherwise it looks like ``machine-config-pools`` is a valid value to give
same goes for the other new parameters

Reply:
OK